### DISCUSSION TITLE
SSO Shown as Open Source feature in docs

### DISCUSSION THREAD

**Discussion by arcarmona** [emqx/emqx#12617] [General]
Hello, SSO is shown as an Open Source feature [here](https://www.emqx.io/docs/en/latest/#product-comparison:~:text=MQTT%20data%20bridge.-,%E2%9C%94%EF%B8%8F%20Audit%20log%20and%20single%20sign%2Don%20(SSO),-%E2%9C%94%EF%B8%8F%20Multi%2Dprotocol%20gateways), however from what's shown in the [SSO page](https://www.emqx.io/docs/en/latest/dashboard/sso.html#single-sign-on-sso) I believe it's an Enterprise one.

**Comment by lafirest** [comment]
@arcarmona 
Thanks for your feedback, I think this is an editing error and we will fix it.
